### ISSUE TITLE
Bundle iD?

### ISSUE THREAD

**Issue by dereklieu** [developmentseed/macrocosm#19]
@olafveerman @kamicut should we bundle a version of iD, perhaps as a submodule? Then we could also include a way to make the alterations to iD that would allow this to run out of the box, as well as include the necessary npm scripts to make it a one-liner.

**Comment by kamicut** [comment]
Yes, I agree, we should fork iD and bundle it for here. However, should we keep in sync with upstream changes or pin to a version?

**Comment by dereklieu** [comment]
I think we should keep it in sync. One way is to fork iD, make changes locally that would make it work with Macrocosm, then include that fork as a submodule. We would then have to maintain the fork and update it with upstream changes when those occur, as needed.

**Comment by dereklieu** [comment]
I realize this makes it sound like the current strategy with OR, but the difference being that we don't need to do a ton of work to make iD work with macrocosm.

**Comment by mapmeld** [comment]
I had an issue with the current git submodule for iD:

``` bash
ubuntu@ip-172-31-27-49:~/macrocosm$ git submodule update --init
Submodule 'iD' (git://github.com/macrocosm/iD.git) registered for path 'iD'
Cloning into 'iD'...
remote: Counting objects: 51378, done.
remote: Compressing objects: 100% (87/87), done.
remote: Total 51378 (delta 45), reused 0 (delta 0), pack-reused 51291
Receiving objects: 100% (51378/51378), 67.50 MiB | 2.91 MiB/s, done.
Resolving deltas: 100% (34846/34846), done.
Checking connectivity... done.
fatal: reference is not a tree: e59165953b6f148656a89969456769d009a77fd1
Unable to checkout 'e59165953b6f148656a89969456769d009a77fd1' in submodule path 'iD'
```

**Comment by dereklieu** [comment]
@mapmeld just pushed an update, submodule points to the right commit now.
